### PR TITLE
Correct display of tooltip for group attributes

### PR DESCRIPTION
**Pull request by Badlapje**
Correct display of tooltip for group attributes

Prior to this change, goup attributes had their tooltip on every row.  The icon for the tooltip was below the value instead of next to it (as for normal attributes) and the tooltip become bold.

This change ensures the tooltip doesn't repeat, the icon is positioned properly and the text of the tooltip is normal weight.

The before screenshot is present in the wiki page mentioned below.  The after screenshot looks like this:
image

Testing can be done by changing line 93-98 of the ConsentController in the functional testing bundle to:
```
            'urn:mace:dir:attribute-def:isMemberOf' => [
                'urn:collab:org:vm.openconext.org',
                'urn:collab:org:vm.openconext.org2:biefstuk:met:frietjes',
                'urn:collab:org:vm.openconext.org3:witloof:in:den:oven',
                'urn:collab:org:vm.openconext.org4:vanillepudding:met:speculoos',
                'urn:collab:org:vm.openconext.org5:ooohooohohoohoooo:n:potje:choco',
                'urn:collab:org:vm.openconext.org6:schatjes:van:patatjes',
                'urn:collab:org:vm.openconext.org8:1:reep:chocola:is:geen:reep:chocola',
            ],
        ];
        $attributeMotivations = [
            'urn:mace:dir:attribute-def:eduPersonPrincipalName' => 'Test  tooltip',
            'urn:mace:dir:attribute-def:givenName' => 'Test tooltip',
            'urn:mace:dir:attribute-def:isMemberOf' => 'Test tooltip'
        ];
```

Issue discovered as part of acceptation testing and [reported on the wiki](https://wiki.surfnet.nl/display/coininfra/Bevindingen+nav+tests+door+supportteam).

### CODE REVIEW

File: theme/base/templates/modules/Authentication/View/Proxy/Partials/Consent/Attributes/defaultAttribute.html.twig [hunk 1]
```diff
@@ -29,7 +29,7 @@
             {% for value in attributeValues %}
                 <li>
                     {% include '@theme/Authentication/View/Proxy/Partials/Consent/Attributes/value.html.twig' with { attributeValue: value } %}
-                    {% if addTooltip %}
+                    {% if addTooltip and loop.first %}
```
Comment (MKodde) [inline theme/base/templates/modules/Authentication/View/Proxy/Partials/Consent/Attributes/defaultAttribute.html.twig:32]: TIL: Twig has a very cool feature `loop.first`!

Comment (Badlapje) [reply theme/base/templates/modules/Authentication/View/Proxy/Partials/Consent/Attributes/defaultAttribute.html.twig:32]: gotta love it 💯

**Review by MKodde — COMMENTED** [review]
A pragmatic solution, levering a cool twig loop feature :+1: 

See my screenshot. Layout might need some TLC?

Screenshot_2021-02-10 OpenConext - Review your information that will be shared


PS, I added a commit to this PR, extending the consent behaviour. Including the multival attribute. Hope this does not FU existing tests :crossed_fingers:

**Comment by Badlapje** [comment]
> See my screenshot. Layout might need some TLC?
> 
> Screenshot_2021-02-10 OpenConext - Review your information that will be shared

I can't replicate that Screenshot?  What browser is it?  

> PS, I added a commit to this PR, extending the consent behaviour. Including the multival attribute. Hope this does not FU existing tests 🤞

i don't see it?

**Comment by MKodde** [comment]
Firefox, probably cached css on my side then. Letmeflushthatrealquick

**Review by MKodde — APPROVED** [review]

**Comment by MKodde** [comment]
Maybe in one fell swoop squash the last two commits. They are very much related to one another, and 'Fix failing text' would not pass Tony's commit rulebook I suppose ;P

I can live with this too, just nit picking here

**Pull request merged**